### PR TITLE
Document API changes from webnative 0.23

### PR DESCRIPTION
**Pull request by matheus23**
* Changed return type of `index`
* Changed `deleteByURL` to `deleteByDomain`
* Added `publish`

### CODE REVIEW

File: developers/webnative/platform.md [hunk 1]
```diff
@@ -44,9 +44,27 @@ const newApp = await sdk.apps.create()
...
+
+Params:
+
+* domain: `string` **required**
+* cid: `string` **required**
+
+Example:
+
+```typescript
+await sdk.apps.publish('your-fission-deployment.fission.app', 'QmRVvvMeMEPi1zerpXYH9df3ATdzuB63R1wf3Mz5NS5HQN')
```
Comment (bgins) [inline developers/webnative/platform.md:59]: Can a user enter anything for the `domain` or does it need to be a domain from `apps.create`?

File: developers/webnative/platform.md [hunk 2]
```diff
@@ -44,9 +44,27 @@ const newApp = await sdk.apps.create()
...
+```
+
+Getting a CID can be tricky. Here's a way to turn a WNFS public subdirectory into a CID:
+https://github.com/fission-suite/dashboard/blob/8ef6e7031b8f6f20f2927e7e6c16349d9ba24ec9/src/Javascript/index.js#L334-L339
```
Comment (bgins) [inline developers/webnative/platform.md]: Wondering if we may want to inline this example code into the docs? It's not really what this section is about, but we can give it more context by putting it here. Like, what is `appPath`? Is the app a public subdirectory?

**Review by bgins — COMMENTED** [review]
Awesome, thanks for updating this section. :pray:  Left some comments on sections that could use clarification.

File: developers/webnative/platform.md (hunk 1 above)
Comment (matheus23) [reply developers/webnative/platform.md:59]: It has to be something that was created before.

`publish` might not be the very best name for this function. It used to be `update`, but that wasn't ideal either.

File: developers/webnative/platform.md (hunk 2 above)
Comment (matheus23) [reply developers/webnative/platform.md]: Yup sounds like a good idea!

**Review by bgins — APPROVED** [review]
Looks good! Thanks again :tada:

**Pull request merged**